src/components/UploadEditor.vue

```diff
@@ -24,38 +24,45 @@
 		class="upload-editor"
 		container="#content-vue"
 		@close="handleDismiss">
-		<!--native file picker, hidden -->
-		<input id="file-upload"
-			ref="fileUploadInput"
-			multiple
-			type="file"
-			class="hidden-visually"
-			@change="handleFileInput">
-		<transition-group
-			class="upload-editor__previews"
-			name="fade"
-			tag="div">
-			<template v-for="file in files">
-				<FilePreview
-					:key="file.temporaryMessage.id"
-					v-bind="file.temporaryMessage.messageParameters.file"
-					:is-upload-editor="true"
-					@remove-file="handleRemoveFileFromSelection" />
-			</template>
-			<div v-if="!isVoiceMessage"
-				:key="'addMore'"
-				class="add-more">
-				<button :aria-label="addMoreAriaLabel"
-					class="add-more__button primary"
-					@click="clickImportInput">
-					<Plus
-						decorative
-						title=""
-						:size="48"
-						class="upload-editor__plus-icon" />
-				</button>
-			</div>
-		</transition-group>
+		<template v-if="!isVoiceMessage">
+			<!--native file picker, hidden -->
+			<input id="file-upload"
+				ref="fileUploadInput"
+				multiple
+				type="file"
+				class="hidden-visually"
+				@change="handleFileInput">
+			<transition-group
+				class="upload-editor__previews"
+				name="fade"
+				tag="div">
+				<template v-for="file in files">
+					<FilePreview
+						:key="file.temporaryMessage.id"
+						v-bind="file.temporaryMessage.messageParameters.file"
+						:is-upload-editor="true"
+						@remove-file="handleRemoveFileFromSelection" />
+				</template>
+				<div
+					:key="'addMore'"
```

Comment:
on a div? what does it do?

Reply:
It's because it's inside a loop so it needs a key

Reply:
but its not a component and div html elements don't have a key attribute?